toolchain/semantics/semantics_handle_operator.cpp

```diff
@@ -61,7 +61,27 @@ auto SemanticsHandleInfixOperator(SemanticsContext& context,
 
 auto SemanticsHandlePostfixOperator(SemanticsContext& context,
                                     ParseTree::Node parse_node) -> bool {
-  return context.TODO(parse_node, "HandlePostfixOperator");
+  auto value_id = context.node_stack().PopExpression();
+
+  // Figure out the operator for the token.
+  auto token = context.parse_tree().node_token(parse_node);
+  switch (auto token_kind = context.tokens().GetKind(token)) {
+    case TokenKind::Star: {
```

Comment:
The switch with one case seems odd to me -- how many postfix operators do you anticipate? My impression was we're dropping `++` and `--` from C++, so maybe `*` will be the only one?

Note, if you're not anticipating others, it might be more correct in the default to CARBON_FATAL instead of TODO. I want to get rid of the TODO once semantics is more complete, it's kind of a patchwork thing while we're building the toolchain so shouldn't be used where we don't anticipate more work.